src/features/events/components/EventParticipantsCard.tsx

```diff
@@ -0,0 +1,221 @@
+                          return;
+                        }
+
+                        const intVal = parseInt(val);
+                        if (!isNaN(intVal) && intVal.toString() == val) {
+                          setNewReqParticipants(intVal);
+                        }
+                      }}
+                      onKeyDown={(ev) => {
+                        if (ev.key === 'Enter') {
+                          setAnchorEl(null);
+                          if (newReqParticipants != null) {
+                            model.setReqParticipants(newReqParticipants);
+                          }
+                        } else if (ev.key === 'Escape') {
+                          setAnchorEl(null);
+                          setNewReqParticipants(reqParticipants);
+                        }
+                      }}
+                      value={
+                        newReqParticipants === null ? '' : newReqParticipants
+                      }
+                      variant="outlined"
+                    />
+                  </Box>
+                </Paper>
+              </ClickAwayListener>
+            </Popper>
+          </Box>
+        }
+      >
+        <Divider />
+        <Box
+          display="flex"
+          flexDirection="column"
+          justifyContent="space-between"
+        >
+          <Box
+            alignItems="center"
+            display="flex"
+            justifyContent="space-between"
+            marginY={1}
+          >
+            <Typography color={'secondary'} component="h6" variant="subtitle1">
+              {messages.eventParticipantsCard.pending()}
+            </Typography>
+            <Typography>
+              {Math.max(reqParticipants - availParticipants, 0)}
+            </Typography>
+          </Box>
+          <Box
+            alignItems="center"
+            display="flex"
+            justifyContent="space-between"
+            marginBottom={1}
+          >
+            <Typography color={'secondary'} component="h6" variant="subtitle1">
+              {messages.eventParticipantsCard.booked()}
+            </Typography>
+            <Typography>{`${availParticipants}/${availParticipants}`}</Typography>
+          </Box>
+          <Box
+            alignItems="center"
+            display="flex"
+            justifyContent="space-between"
+            marginBottom={1}
+          >
+            <Typography color={'secondary'} component="h6" variant="subtitle1">
+              {messages.eventParticipantsCard.contact()}
+            </Typography>
+            {contact && eventData.contact && (
+              <Box alignItems="center" display="flex">
+                <Avatar
+                  src={`/api/orgs/${eventData.organization.id}/people/${eventData.contact.id}/avatar`}
+                  style={{ height: 30, marginRight: 10, width: 30 }}
+                />
+                <ZUIPersonLink person={contact} />
+              </Box>
+            )}
```

Comment:
I'm sorry but I think @ziggabyte and I may have lead you down a wrong path here. I think a better solution to using an avatar and a `ZUIPersonLink` would have been to just wrap what you had before in a `ZUIHoverCard`, that way you would get the same hover card as now, but without the extra styling and the need to retrieve more data first.